src/main/java/uk/ac/ebi/eva/contigalias/controller/contigalias/ContigAliasHandler.java

```diff
@@ -91,229 +88,85 @@ public PagedModel<EntityModel<AssemblyEntity>> getAssembliesByTaxid(long taxid,
 
     public PagedModel<EntityModel<AssemblyEntity>> getAssembliesBySequenceGenbank(String genbank) {
         List<AssemblyEntity> assemblies = chromosomeService.getAssembliesByChromosomeGenbank(genbank);
-        assemblies.addAll(scaffoldService.getAssembliesByScaffoldGenbank(genbank));
         return generatePagedModelFromPage(new PageImpl<>(assemblies), assemblyAssembler);
     }
 
     public PagedModel<EntityModel<AssemblyEntity>> getAssembliesBySequenceRefseq(String refseq) {
         List<AssemblyEntity> assemblies = chromosomeService.getAssembliesByChromosomeRefseq(refseq);
-        assemblies.addAll(scaffoldService.getAssembliesByScaffoldRefseq(refseq));
         return generatePagedModelFromPage(new PageImpl<>(assemblies), assemblyAssembler);
     }
 
     public PagedModel<EntityModel<SequenceEntity>> getSequencesByGenbank(String genbank, Pageable request) {
-        long count = chromosomeService.countChromosomeEntitiesByGenbank(genbank);
-        List<Pageable>[] pageRequests = createScaffoldsPageRequest(count, request);
-        List<Page<? extends SequenceEntity>> pages = new LinkedList<>();
-        for (Pageable pageable : pageRequests[0]) {
-            pages.add(chromosomeService.getChromosomesByGenbank(genbank, pageable));
-        }
-        for (Pageable pageable : pageRequests[1]) {
-            pages.add(scaffoldService.getScaffoldsByGenbank(genbank, pageable));
-        }
-        return generatePagedModelFromPage(createSequencePage(pages), sequenceAssembler);
+        Page<ChromosomeEntity> chrPage = chromosomeService.getChromosomesByGenbank(genbank, request);
+        return generatePagedModelFromPage(createSequencePage(chrPage), sequenceAssembler);
```

Comment:
Is there still a use case for making chromosomes accessible separately from other scaffolds at the API level?  Can be another ticket but just seems like it might be nice especially in cases where there are thousands of scaffolds and just a couple dozen chromosomes...

Reply:
it is still doable "after the fact" by filtering via the new contig_type attribute. but we might have to discuss the specific use-case we are looking at and how prevalent that might be... I will slate this for stand-up.